std_msgs/Header header
uint32 label_id
uint32 instance_id
geometry_msgs/Vector3 centroid
geometry_msgs/Vector3 dimensions
sensor_msgs/PointCloud2 pc
